cmake/AutodetectEditbin.cmake

```diff
@@ -0,0 +1,29 @@
+# Autodetect editbin. Only useful for MSVC.
+#
+# autodetect_editbin()
+#
+function(autodetect_editbin)
+    if (DEFINED EDITBIN_FOUND)
+        return()
+    endif (DEFINED EDITBIN_FOUND)
+
+    message(STATUS "Detecting editbin")
+    find_program(EDITBIN_EXECUTABLE editbin.exe)
+
+    if (EDITBIN_EXECUTABLE STREQUAL "EDITBIN_EXECUTABLE-NOTFOUND")
```

Comment:
```
if (NOT EDITBIN_EXECUTABLE)
```
But I would always calculate the MSVC path and just pass it to find_program() in HINTS.